client/webpack.client.rails.build.config.js

```diff
@@ -32,7 +35,7 @@ module.exports = merge(config, {
       'bootstrap-loader/extractStyles'
     ],
   },
-
+  mode: process.env.NODE_ENV,
```

Comment:
Remove this and add
```
mode: process.env.NODE_ENV === 'development' ? 'development' : 'production',
```
to `client/webpack.client.base.config.js` & `client/webpack.server.rails.build.config.js`